Packs/DemistoRESTAPI/Scripts/GetIncidentTasks/GetIncidentTasks.py

```diff
@@ -0,0 +1,167 @@
+        tag (str | None): the tag to compar with the task or None, if not searching by tag
+        states (list[str]): a list of states that the the task may have
+
+    Returns:
+        bool: `True` if the task matches the given name, tag an/or state, otherwise `False`
+    """
+    task_task = task.get('task', {})
+    name_match = name is None or task_task.get('name', '').lower() == name.lower()
+    tag_match = tag is None or tag in task_task.get('tags', [])
+    state_match = len(states) == 0 or task.get('state') in states
+
+    return name_match and tag_match and state_match
+
+
+def format_title(name: str | None, tag: str | None, states: str) -> str:
+    """_summary_
+
+    Args:
+        name (str | None): The name of the task
+        tag (str | None): The tag assigned to the task
+        states (str): A list of possible states that the task can have
+
+    Returns:
+        str: Formatted title for the war room
+    """
+    items: list[str] = []
+    if name is not None:
+        items.append(f'name "{name}" ')
+    if tag:
+        items.append(f'tag "{tag}" ')
+    if states:
+        items.append(f'states "{states}" ')
+    return f'Tasks with {"and ".join(items)}'.strip()
+
+
+''' COMMAND FUNCTION '''
+
+
+def get_task_command(args: dict[str, Any]) -> CommandResults:
+    name = args.get('name')
+    tag = args.get('tag')
+    states = get_states(argToList(args.get('states')))
+    inc_id = args['inc_id']  # pylint: disable=W9019
+    res = demisto.executeCommand('core-api-get', {'uri': f'/investigation/{inc_id}/workplan'})
+    if not res or isError(res[0]):
+        raise Exception(res)
+
+    tasks: dict = dict_safe_get(res[0], ['Contents', 'response', 'invPlaybook', 'tasks'], {})
+    if not tasks:
+        return CommandResults(readable_output=f'Workplan for incident {inc_id} has no tasks.')
+
+    allTasks = get_playbook_tasks(list(tasks.values()))
+    res = []
+
+    for task in allTasks:
+        if is_task_match(task, name, tag, states):
+            res.append({
+                'id': task.get('id'),
+                'name': task.get('task', {}).get('name', ''),
+                'type': task.get('type'),
+                'owner': task.get('assignee'),
+                'state': task.get('state'),
+                'scriptId': task.get('task', {}).get('scriptId'),
+                'startDate': task.get('startDate'),
+                'dueDate': task.get('dueDate'),
+                'completedDate': task.get('completedDate'),
+                'parentPlaybookID': task.get('parentPlaybookID'),
+                'completedBy': task.get('completedBy')
+            })
+
+    return CommandResults(outputs_prefix='Tasks',
+                          outputs_key_field='id',
+                          outputs=res,
+                          readable_output=tableToMarkdown(
+                              f'{format_title(name, tag,args.get("states", ""))}(Incident  # {inc_id})',
```

Comment:
```suggestion
                               f'Incident #{inc_id} Playbook Tasks',
```
Let's use this format to comply with current conventions.